app/models/miq_request.rb

```diff
@@ -166,8 +168,8 @@ def call_automate_event(event_name)
     MiqAeEvent.raise_evm_event(event_name, self, build_request_event(event_name))
     _log.info("Raised  event [#{event_name}] to Automate")
   rescue MiqAeException::Error => err
-    message = _("Error returned from %{name} event processing in Automate: %{error_message}") %
-                {:name => event_name, :error_message => err.message}
+    _("Error returned from %{name} event processing in Automate: %{error_message}") %
+      {:name => event_name, :error_message => err.message}
```

Comment:
These lines are not being used the way the method is written today.  Thinking we should log this error message and leave the `raise` line alone to re-raise the initial error.  Thoughts?